content/en/docs/concepts/services-networking/service.md

```diff
@@ -917,6 +931,31 @@ Service is a top-level resource in the Kubernetes REST API. More details about t
 API object can be found at:
 [Service API object](/docs/reference/generated/kubernetes-api/{{< param "version" >}}/#service-v1-core).
 
+## SCTP support
+
+{{< feature-state for_k8s_version="v1.12" state="alpha" >}}
+
+Kubernetes introduces the support of SCTP as a `protocol` value in `Service`, `Endpoint`, `NetworkPolicy` and `Pod` definitions as an alpha feature. To enable this feature, the cluster administrator needs to enable the `SCTPSupport` feature gate on the apiserver, for example, `“--feature-gates=SCTPSupport=true,...”`. When the feature gate is enabled, users can set the `protocol` field of a `Service`, `Endpoint`, `NetworkPolicy` and `Pod` to `SCTP`. Kubernetes sets up the network accordingly for the SCTP associations, just like it does for TCP connections.
+
+The kube-proxy sets up iptables rules for some `Service` types: `Service` with `clusterIP`, `Service` with `nodePort`, and `Service` with `externalIP`. In order to support proper iptables based redirection the following kernel modules shall be loaded on the host: `nf_conntrack_proto_sctp` and `nf_nat_proto_sctp`.   
```

Comment:
iptables is an implementation detail.   If we want to talk about this at all, we need to talk about the differences between iptables mode and ipvs mode (did we test IPVS mode?)

Reply:
Right, it is a bit complex topic (especially considering those cases when the kube-proxy starts managing iptables rules in ipvs mode, too). I delete this part.
We have not tested ipvs mode yet e2e, only unit tests run.

Reply:
But have we verified SCTP with IPVS in general?

Reply:
I have created a single-master cluster in GCE based on Ubuntu 18.04 LTS. I used the new k8s images that contain SCTP support. I set the proxy-mode of kube-proxy to "ipvs" and then I created an SCTP Service + 2 busybox backend pods. ipvsadm showed the mapping of the ClusterIP to the backend IPs for SCTP like: 
```shell
root@instance-2:~# ipvsadm
IP Virtual Server version 1.2.1 (size=4096)
Prot LocalAddress:Port Scheduler Flags
  -> RemoteAddress:Port           Forward Weight ActiveConn InActConn
TCP  instance-2:https rr
  -> instance-2.c.k8s-sctp-test.i Masq    1      3          0
TCP  instance-2:domain rr
  -> 10.244.0.2:domain            Masq    1      0          0
  -> 10.244.0.3:domain            Masq    1      0          0
UDP  instance-2:domain rr
  -> 10.244.0.2:domain            Masq    1      0          0
  -> 10.244.0.3:domain            Masq    1      0          0
SCTP instance-2:7777 rr
  -> 10.244.0.4:7777              Masq    1      0          0
  -> 10.244.0.5:7777              Masq    1      0          0
```